src/hmem_cuda.c

```diff
@@ -582,6 +582,9 @@ int cuda_host_unregister(void *ptr)
 
 bool cuda_is_ipc_enabled(void)
 {
+	if (ofi_hmem_p2p_disabled())
+		return false;
+
```

Comment:
can simply as return !ofi_hmem_p2p_disabled() && cuda_ipc_enabled

Reply:
Sorry about that, updated.